gamebook/tests/test_scene.py

```diff
@@ -58,24 +61,27 @@ def test_draw():
 
 def test_draw_when_options_is_empty():
 
-    scene1_name = "scene1"
+    example_scene = Scene("exapmle scene",
+                          desc="this is an example scene",
+                          options=[])
 
-    scene1 = Scene(scene1_name, "welcome", [])
+    with pytest.raises(ValueError) as error_text:
+        assert example_scene.next_scene_draw("n")
```

Comment:
What is the check here?

Reply:
it checks if the ValueError is raised and the correct one without causing the test to fail like before when i ran it

Reply:
this is where i learned it
[https://dev.to/wangonya/asserting-exceptions-with-pytest-8hl](url)

Reply:
ok